docs/volcano_integration.md

```diff
@@ -0,0 +1,202 @@
+# Integration with Volcano for Batch Scheduling
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workloads.
+With the integration with Volcano, Flink job and task managers can be scheduled simultaneously, which is particularly suitable for 
+clusters with resource shortage.
+
+## Prerequisites
+
+## Install Volcano
+
+- Install from provided demo
+
+Run the following 
+```bash
+kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
+```
+
+- Install with advanced settings
+
+Please refer to [Volcano Official Guide](https://volcano.sh/docs/getting-started/)
+
+### Verify Volcano is up and run
+
+```bash
+$ kubectl get all -n volcano-system
```

Comment:
The output of this command it too long, maybe just briefly mention what to check in the output.